packages/gatsby-plugin-stylus/README.md

```diff
@@ -1,3 +1,56 @@
 # gatsby-plugin-stylus
 
-Stub README
+Provides drop-in support for Stylus with or without CSS Modules
+
+## Install
+
+`yarn add gatsby-plugin-stylus`
+
+## How to use
+
+1. Include the plugin in your `gatsby-config.js` file.
+2. Write your stylesheets in Stylus (`.styl` files) and require/import them
+
+### Without CSS Modules
+
+```javascript
+// in gatsby-config.js
+plugins: [
+  `gatsby-plugin-stylus`
+]
+```
+
+### With CSS Modules
```

Comment:
OK, yeah I'm all for consistency. I thought about this particular issue for a bit and came to the conclusion that I've never once had a case where I wanted file-by-file control over whether or not to use CSS modules. It was always based on file type. I.e. I would not use modules with external CSS files but would use it for all my own internal stylus.

Anyway, like I said I'm all for staying consistent with the convention, but I still like the option to enable modules for the entire file type. I will see about adding both.